http-clients/http-clients.html

```diff
@@ -0,0 +1,20 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title>getandtakehttpclients</title>
+</head>
+<body>
+    <script>
+        const apiUrl = "https://localhost:7167/api";
+        fetch(apiUrl + '/products')
```

Comment:
Just an advice: usage of string interpolation is more preferable for JS, so, your code will look like
```suggestion
        fetch(`${apiUrl}/products`)
```